config/software/ruby-windows-devkit-bash.rb

```diff
@@ -18,6 +18,8 @@
 name "ruby-windows-devkit-bash"
 default_version "3.1.23-4-msys-1.0.18"
 
+license :project_license
```

Comment:
According to the [release notes](https://github.com/chef/msys-bash/blob/master/msys-bash.RELEASE_NOTES#L63) which is pointing to http://www.gnu.org/software/bash/ license should be GPL-3.0:

```